mlir/lib/Mitigation/Transforms/MitigationMethods/Zne.cpp

```diff
@@ -45,6 +45,9 @@ void ZneLowering::rewrite(mitigation::ZneOp op, PatternRewriter &rewriter) const
     RankedTensorType scaleFactorType = scaleFactors.getType().cast<RankedTensorType>();
     const auto sizeInt = scaleFactorType.getDimSize(0);
 
+    // Folding type
+    auto folding = op.getFolding();
```

Comment:
The C++ compiler will probably raise a warning about the unused variable, maybe this avoids it:
```suggestion
    op.getFolding();
```

Reply:
Is it necessary to be added in this PR?

Reply:
No, it was a leftover from debugging. I will remove as this belongs to a follow-up PR.